backend/automatic_walk_time_tables/utils/point.py

```diff
@@ -5,9 +5,9 @@
 
 class PointType:
     NONE = "NONE"
-    LV03 = "LV03"
-    LV95 = "LV95"
-    WGS84 = "WGS84"
+    LV03 = "21781"
+    LV95 = "2056"
+    WGS84 = "4979"
```

Comment:
We should add a comment about where these numbers come from: see https://en.wikipedia.org/wiki/EPSG_Geodetic_Parameter_Dataset